vendor/k8s.io/system-validators/validators/types_unix.go

```diff
@@ -59,7 +59,7 @@ var DefaultSysSpec = SysSpec{
 	Cgroups: []string{"cpu", "cpuacct", "cpuset", "devices", "freezer", "memory"},
 	RuntimeSpec: RuntimeSpec{
 		DockerSpec: &DockerSpec{
-			Version:     []string{`1\.1[1-3]\..*`, `17\.0[3,6,9]\..*`, `18\.0[6,9]\..*`, `19\.03\..*`},
+			Version:     []string{`1\.1[1-3]\..*`, `17\.0[3,6,9]\..*`, `18\.0[6,9]\..*`},
```

Comment:
is this change expected?

Reply:
@neolit123 It's not expected, we diverged here too.

 Thanks for the catch @BenTheElder !

Reply:
fixed in 1.0.3

Reply:
it's not https://github.com/kubernetes/system-validators/blob/master/validators/types_unix.go#L62

I'm opening a PR to fix it

Reply:
one patch release per commit, until we get it right :this_is_fine:

Reply:
fixed in 1.0.4

Reply:
Ack